src/lambda/local/debugConfiguration.ts

```diff
@@ -80,7 +80,10 @@ export function makeCoreCLRDebugConfiguration({
         name: 'SamLocalDebug',
         type: 'coreclr',
         request: 'attach',
-        processId: '1',
+        // Since SAM CLI 1.0 we cannot assume PID=1. So use processName=dotnet
```

Comment:
Are the changes to this file backwards compatible with using SAM CLI versions earlier than 1.0?

Reply:
With sam `0.45.0`, using `Debug Locally` pulls Docker image `lambci/lambda:dotnetcore2.1`, the debugger attaches and breakpoints are successfully hit in vscode.